.cordova/hooks/after_prepare/version.js

```diff
@@ -42,6 +42,16 @@
         'config.xml'
     ),
     version = shell.exec('git describe --tags', {silent:true}).output.trim();
+  var etSubElement = et.SubElement;
+
+  var displayName = projectName;
+  if (require(projectConfigFilePath).displayName) {
```

Comment:
Might make the code a little cleaner to do a "configFile = require(projectConfigFilePath)" rather than repeating the latter several times.